android/src/main/kotlin/com/posthog/posthog_flutter/PosthogFlutterPlugin.kt

```diff
@@ -53,6 +59,9 @@ class PosthogFlutterPlugin : FlutterPlugin, MethodCallHandler {
                 debug = enableDebug
                 sdkName = "posthog-flutter"
                 sdkVersion = postHogVersion
+                onFeatureFlags = PostHogOnFeatureFlags {
```

Comment:
Using the callback is a good idea but it'd not work in some cases, for example
https://github.com/PostHog/posthog-android/blob/8013cccadfe52ad347eedcb492ce01690a1f7786/posthog/src/main/java/com/posthog/internal/PostHogFeatureFlags.kt#L40-L45
in this case, the callback will never be called, we'd need to either change the `PostHogOnFeatureFlags` interface which is a breaking change or find a way to expose the loaded flags differently (on the iOS and Android SDK)